query-engine/query-engine-c-abi/copy-android.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+TARGET_DIR=../../../react-native-prisma
```

Comment:
yeah, the resulting binaries are (will be) embedded in an npm downloadable library for react native devs. We can make this configurable, but since the focus is on RN for now, I think we can postpone this for later.